test/tutorial007/expected

```diff
@@ -0,0 +1 @@
+sizeof(size_t): 8 bytes
```

Comment:
This looks like a problematic assumption. Tests are supposed to work everywhere.